Polish SDK 4.0 docs - Closes #736 (#737)
LiskHQ/lisk-docs: modules/ROOT/pages/tutorials/hello-world.adoc

<2> The transaction is signed by the genesis account.

> <3> The transaction is displayed as JSON object in the console.

Reviewer: ```suggestion
<3> The transaction is displayed as a JSON object in the console.
```

[TIP]
====

> To get the nonce of an account, look at the respective account details.

Reviewer: ```suggestion
To get the nonce of an account, view the respective account details.
```

> You can do this via API call like described in the guide xref:{url_guide_broadcast}[Broadcast a transaction].

Reviewer: ```suggestion
This can be achieved via an API call as described in the guide xref:{url_guide_broadcast}[Broadcast a transaction].
```
